fastchat/model/model_adapter.py

```diff
@@ -1147,6 +1147,22 @@ def load_model(self, model_path: str, from_pretrained_kwargs: dict):
     def get_default_conv_template(self, model_path: str) -> Conversation:
         return get_conv_template("internlm-chat")
 
+class OrcaAdapter(BaseModelAdapter):
+    """The model adapter for psmathur/orca_mini_v2_7b"""
+
+    def match(self, model_path: str):
+        return "orca" in model_path
+
+    def load_model(self, model_path: str, from_pretrained_kwargs: dict):
+        model = LlamaForCausalLM.from_pretrained(
+                model_path, torch_dtype=torch.float16, device_map='auto',
```

Comment:
1. I think we should add `low_cpu_mem_usage=True` to minimize the RAM usage when loading the model.
2. I think we should also add `**from_pretrained_kwargs`, the `torch_dtype` and `device_map` should be passed by `from_pretrained_kwargs` instead of forcing it.